inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/channel/TestBufferQueueChannel.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.dataproxy.channel;
+
+import org.apache.flume.Channel;
+import org.apache.flume.Context;
+import org.apache.flume.Transaction;
+import org.apache.flume.channel.MemoryChannel;
+import org.apache.flume.event.SimpleEvent;
+import org.junit.Test;
+
+/**
+ * BufferQueueChannelTest
+ * 
+ */
+public class TestBufferQueueChannel {
+
+    /**
+     * testResult
+     * MemoryChannel,times:1000000,transactionSize:1,duration:1363<br>
+     * BufferQueueChannel,times:1000000,transactionSize:1,duration:84,gap:0.9383712399119589<br>
+     * MemoryChannel,times:1000000,transactionSize:10,duration:2818<br>
+     * BufferQueueChannel,times:1000000,transactionSize:10,duration:122,gap:0.9567068843151171<br>
+     * MemoryChannel,times:1000000,transactionSize:100,duration:19189<br>
+     * BufferQueueChannel,times:1000000,transactionSize:100,duration:687,gap:0.9641982385741831<br>
+     * @throws Exception
+     */
+    @Test
+    public void testResult() throws Exception {
+        // init
+        Context memoryContext = new Context();
+        memoryContext.put("capacity", "50000");
+        memoryContext.put("keep-alive", "0");
+        memoryContext.put("transactionCapacity", "200");
+        final MemoryChannel memoryChannel = new MemoryChannel();
+        memoryChannel.configure(memoryContext);
+        memoryChannel.start();
+        // init
+        Context bufferContext = new Context();
+        bufferContext.put("maxBufferQueueCount", "50000");
+        bufferContext.put("maxBufferQueueSizeKb", "50000");
+        final BufferQueueChannel bufferChannel = new BufferQueueChannel();
+        bufferChannel.configure(bufferContext);
+        bufferChannel.start();
+        // times
+        int times = 1000000;
+        // transactionSize=1
+        int transactionSize = 1;
+        long memoryDuration = runChannel(memoryChannel, times, transactionSize);
+        System.out.println(String.format("MemoryChannel,times:%d,transactionSize:%d,duration:%d", times,
```

Comment:
Suggested using log framework to print the log info.

Reply:
fixed